xml/System.ComponentModel/SortDescription.xml

```diff
@@ -39,7 +39,7 @@
         <Parameter Name="direction" Type="System.ComponentModel.ListSortDirection" />
       </Parameters>
       <Docs>
-        <param name="propertyName">The name of the property to sort the list by.</param>
+        <param name="propertyName">The name of the property to sort the list by. Should be set to "" if the collection to be sorted contains strings, integers, etc..</param>
```

Comment:
> Should be set to "" if the collection to be sorted contains strings, integers, etc.

Should be set to empty string if the collection to be sorted contains types like string or numeric types.

Reply:
Agreed.

Reply:
```suggestion
        <param name="propertyName">The name of the property to sort the list by. If the collection contains strings or value types, set this parameter to the empty string (`""`).</param>
```